src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const splitedObj = sourceString
+    .split(';')
+    .map((el) => el.trim())
+    .filter((el) => el !== '');
+
+  const resultObj = {};
```

Comment:
[CODE STYLE] - Avoid variable namings like `resultObj`. It will decrease code readability. Consider using a more descriptive name that reflects the nature of the variable, such as `styleObject` or `cssProperties`.